### PR TITLE
Erdikose/housekeeping/remove settings plugins

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/molecules/NotificationMarkdown/TelemetryButtons.tsx [hunk 1]
```diff
@@ -28,8 +26,11 @@ export const TelemetryButtons = ({notificationId}: {notificationId?: string}) =>
   const handleNotOk = () => {
     disableTelemetryNotification();
 
+    /* 
+    Will not open any where since SettingsPanel is available after opening a project
```
Comment (erdkse) [inline src/components/molecules/NotificationMarkdown/TelemetryButtons.tsx:30]: @devcatalin @topliceanurazvan @manoukyanmari @yasarkubeshop 

Please have a look here

Comment (topliceanurazvan) [reply src/components/molecules/NotificationMarkdown/TelemetryButtons.tsx:30]: Is there a way to check if a project is already opened and only then set the left menu to settings?

File: src/components/organisms/HotKeysHandler/HotKeysHandler.tsx [hunk 2]
```diff
@@ -95,7 +97,8 @@ const HotKeysHandler = () => {
   );
 
   useHotkeys(hotkeys.TOGGLE_SETTINGS.key, () => {
```
Comment (topliceanurazvan) [inline src/components/organisms/HotKeysHandler/HotKeysHandler.tsx:99]: Let's rename this from `TOGGLE_SETTINGS` to `OPEN_SETTINGS`

File: src/components/organisms/HotKeysHandler/HotKeysHandler.tsx [hunk 3]
```diff
@@ -95,7 +97,8 @@ const HotKeysHandler = () => {
...
-    dispatch(toggleSettings());
+    dispatch(setActiveSettingsPanel(SettingsPanel.ActiveProjectSettings));
```
Comment (topliceanurazvan) [inline src/components/organisms/HotKeysHandler/HotKeysHandler.tsx:100]: Why should we select the active settings pane here? What is the first panel inside settings that is selected when you open it?

File: src/components/organisms/PageHeader/ClusterSelection.tsx [hunk 4]
```diff
@@ -69,7 +69,7 @@ const ClusterSelection = ({previewResource}: {previewResource?: K8sResource}) =>
 
   const handleClusterConfigure = () => {
     dispatch(highlightItem(HighlightItems.CLUSTER_PANE_ICON));
-    dispatch(toggleSettings());
```
Comment (topliceanurazvan) [inline src/components/organisms/PageHeader/ClusterSelection.tsx:72]: As above, should we select a specific settings panel for cluster configure?

File: src/components/organisms_new/SettingsPane/types.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export enum SettingsPanel {
```
Comment (topliceanurazvan) [inline src/components/organisms_new/SettingsPane/types.ts:1]: Let's not forget to move this to `@models` after we merge to main

File: src/navsections/K8sResourceSectionBlueprint/K8sResourceSectionEmptyDisplay.tsx [hunk 6]
```diff
@@ -46,7 +48,8 @@ function K8sResourceSectionEmptyDisplay() {
 
     setTimeout(() => {
       if (itemToHighlight === HighlightItems.BROWSE_TEMPLATES) {
-        dispatch(setLeftMenuSelection('templates-pane'));
+        dispatch(setActiveSettingsPanel(SettingsPanel.Plugins));
```
Comment (topliceanurazvan) [inline src/navsections/K8sResourceSectionBlueprint/K8sResourceSectionEmptyDisplay.tsx:51]: I think that templates will have a different pane, not the plugins from settings

File: src/navsections/K8sResourceSectionBlueprint/K8sResourceSectionEmptyDisplay.tsx [hunk 7]
```diff
@@ -61,7 +64,7 @@ function K8sResourceSectionEmptyDisplay() {
 
   const handleClusterConfigure = () => {
     dispatch(highlightItem(HighlightItems.CLUSTER_PANE_ICON));
-    dispatch(toggleSettings());
```
Comment (topliceanurazvan) [inline src/navsections/K8sResourceSectionBlueprint/K8sResourceSectionEmptyDisplay.tsx:64]: Same here, should we select some specific settings panel first?

**Comment by devcatalin** [comment]
Let's close this one and redo the work in a separate PR as this one is stale and has too many conflicts at this point..

**Pull request closed without merging**